tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -13310,6 +14316,32 @@ bool Sema::DiagnoseHLSLDecl(Declarator &D, DeclContext *DC, Expr *BitWidth,
       }
       pMeshModifier = pAttr;
       break;
+    case AttributeList::AT_HLSLNodeDispatchGrid:
+      if (pDispatchGrid) {
+        // TODO: it would be nice to diffentiate between an exact duplicate and
```

Comment:
Do we have an issue tracking this?

Reply:
I have added this to issue https://github.com/microsoft/DirectXShaderCompiler/issues/5699.